scripts/report_event.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+###############################################################################
+# vim: tabstop=4:shiftwidth=4:expandtab:
+# Copyright (c) 2017 SIOS Technology Corp. All rights reserved.
+#
+# Arguments:
+#   args[1] - Windows Event Source/Provider 
+#   args[2] - Windows Event ID
+#   args[3] - Windows Event Severity
+#   args[4] - Windows Event Message
+#   args[5] - Windows Event Time Generated in ISO 8601 format (2017-10-11T15:18:33-0500)
+##############################################################################
+"""
+This script will send three iQ events (one for each layer type) for a single Windows event.
+"""
+
+import logging
```

Comment:
Rm any unused packages

Reply:
Good point. When I moved the timezone handling to powershell I left the pytz and time libraries in the python script. Ok.